spec/pagseguro/transaction_spec.rb

```diff
@@ -61,32 +56,46 @@
     end
   end
 
+  describe ".find_by_reference" do
+    it 'initializes search with given reference code' do
+      now = Time.now
+      Time.stub now: now
+
+      PagSeguro::SearchByReference
+        .should_receive(:new)
```

Comment:
this is the old rspec syntax. for now it's ok, but we must use new syntax in the near future.